### PR TITLE
feature: add convert office files to pdf

### PR DESCRIPTION
**Pull request by csabaujvari**

### CODE REVIEW

File: office.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	req.values[pdfFormatOffice] = pdfFormat
+}
+
+// Merge sets merge form field.
+func (req *OfficeRequest) Merge(merge bool) {
+	req.values[mergeOffice] = strconv.FormatBool(merge)
+}
+
+func (req *OfficeRequest) postURL() string {
+	return "/forms/libreoffice/convert"
```
Comment (HiperJava) [inline office.go:62]: const v config?

File: office.go [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+}
+
+func (req *OfficeRequest) formFiles() map[string]Document {
+	files := make(map[string]Document)
+	for _, doc := range req.docs {
+		files[doc.Filename()] = doc
```
Comment (HiperJava) [inline office.go:68]: Filename -»FileName

**Comment by mateilles718** [comment]
LGTM

**Pull request merged**